.flake8

```diff
@@ -0,0 +1,17 @@
+[flake8]
+ignore=E501
+max-complexity = 100
+exclude =
```

Comment:
I think `.tox`, `__pycache__` and `.git` folders make sense. I had issues with `.git` earlier! :unamused: 
See the configuration given as an example in flake8 docs:

```ini
[flake8]
ignore = D203
exclude =
    .git,
    __pycache__,
    docs/source/conf.py,
    old,
    build,
    dist
max-complexity = 10
```
However, I'm not positive that `*.pyc` → `*.js` are needed, since you had `filename = *.py` in the config (and I'm pretty sure it's flake8's default anyway).